### PR TITLE
[13.0] [MIG] event_sale_session: Migration to v13.0

### PR DESCRIPTION
**Pull request by CarlosRoca13**
cc @Tecnativa TT29929

please @chienandalu @victoralmau review this

### CODE REVIEW

File: event_sale_session/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# Copyright 2017-19 Tecnativa - David Vidal
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+{
+    "name": "Event Sale Sessions",
+    "summary": "Sessions sales in events",
+    "version": "13.0.1.0.0",
+    "author": "Tecnativa, " "Odoo Community Association (OCA)",
```
Comment (victoralmau) [inline event_sale_session/__manifest__.py]:
```suggestion
    "author": "Tecnativa, Odoo Community Association (OCA)",
```

File: event_sale_session/i18n/es.po [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+#~ "Edit them so you can save the sale order"
+#~ msgstr ""
+#~ "Estás intentando reservar en una sesión sin plazas.\n"
+#~ "Edítal para que puedas guardar tu pedido."
+
+#~ msgid "registration.editor"
+#~ msgstr "registration.editor"
+
+#~ msgid "registration.editor.line"
+#~ msgstr "registration.editor.line"
```
Comment (victoralmau) [inline event_sale_session/i18n/es.po]:
```suggestion
```

File: event_sale_session/models/sale_order.py [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+            sessions.setdefault(line.session_id, line.event_session_seats_available)
+            sessions[line.session_id] -= line.product_uom_qty
+            # Break if any session can't allocate seats
+            if sessions[line.session_id] < 0:
+                return False
+        return True
+
+
+class SaleOrderLine(models.Model):
+
```
Comment (victoralmau) [inline event_sale_session/models/sale_order.py]:
```suggestion
```

File: event_sale_session/models/sale_order.py [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+    _inherit = "sale.order.line"
+
+    session_id = fields.Many2one(comodel_name="event.session", string="Session",)
```
Comment (victoralmau) [inline event_sale_session/models/sale_order.py]:
```suggestion
    session_id = fields.Many2one(comodel_name="event.session", string="Session")
```

File: event_sale_session/tests/test_event_sale_session.py [hunk 5]
```diff
@@ -0,0 +1,152 @@
...
+        sale.action_confirm()
+        self.assertEqual(self.session.unconfirmed_qty, 0)
+        self.assertEqual(self.event.unconfirmed_qty, 0)
+        regs = self.env["event.registration"].search([("sale_order_id", "=", sale.id)])
+        self.assertTrue(len(regs) > 0)
+        for reg in regs:
+            self.assertEqual(reg.event_id.id, self.event.id)
+            self.assertEqual(reg.session_id.id, self.session.id)
+            self.assertEqual(reg.partner_id.id, self.partner.id)
+            self.assertEqual(reg.name, self.partner.name)
```
Comment (victoralmau) [inline event_sale_session/tests/test_event_sale_session.py:109]:
```suggestion
        self.assertEqual(len(regs), 1)
        self.assertEqual(regs.event_id.id, self.event.id)
        self.assertEqual(regs.session_id.id, self.session.id)
        self.assertEqual(regs.partner_id.id, self.partner.id)
        self.assertEqual(regs.name, self.partner.name)
```

File: event_sale_session/wizard/event_configurator.py [hunk 6]
```diff
@@ -0,0 +1,19 @@
+# Copyright 2021 Tecnativa - Carlos Roca
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class EventConfigurator(models.TransientModel):
+    _inherit = "event.event.configurator"
+
+    session_id = fields.Many2one("event.session", string="Session")
```
Comment (victoralmau) [inline event_sale_session/wizard/event_configurator.py]:
```suggestion
    session_id = fields.Many2one(comodel_name="event.session", string="Session")
```

**Review by victoralmau — CHANGES_REQUESTED** [review]
Good job, only some little changes.

**Comment by CarlosRoca13** [comment]
@victoralmau changes done

File: event_sale_session/tests/test_event_sale_session.py (hunk 5 above)
Comment (CarlosRoca13) [reply event_sale_session/tests/test_event_sale_session.py:109]: I'm going to recover this, because there are more than 1 record

**Review by victoralmau — APPROVED** [review]

**Review by joao-p-marques — APPROVED** [review]
Code review :+1:

**Pull request merged**